crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip.rs

```diff
@@ -50,7 +50,8 @@ mod store;
 mod bandwidth;
 mod next_target;
 
-#[cfg(all(test, feature = "test_utils"))]
+#[cfg(feature = "test_utils")]
```

Comment:
Shouldn't this be any?

Reply:
@freesig yeah, we definitely only want the tests to be compiled when we're running tests. And also the tests can't run without the test utils, so it needs to be both.

Reply:
I don't think this means that. `test_utils` are only needed because `cfg(test)` doesn't transfer to  dep crates. So `test_utils` always implies `test`. But this makes it awkward when you are actually working on the crate because it means you can't do `cargo test` and you must pass `test_utils` to the feature flag which means we end up missing tests.